osv/ecosystems/_ecosystems.py

```diff
@@ -130,6 +127,12 @@ def get(name: str) -> Ecosystem:
   if name.startswith('Ubuntu'):
     return Ubuntu()
 
+  if name.startswith('openSUSE'):
+    return OpenSUSE()
+
+  if name.startswith('SUSE'):
```

Comment:
nit: I think these checks should include the `:`, since it seems likes it's required.

Reply:
If we include `:`, then all the API quires will have to include the release name such as "SUSE:Linux Enterprise Software Development Kit 11 SP4". I think it's not ideal.